tools/src/launcher/Main.hs

```diff
@@ -133,6 +136,9 @@ optionsParser = do
         short   'w' <>
         help    "An argument to be passed to the wallet frontend executable." <>
         metavar "ARG"
+    loWalletLogging <- switch $
+        long    "wlogging" <>
+        help    "Logging flag for the launcher wallet."
```

Comment:
Logging flag what? :)
Does it turn on the logging? From Info to Debug? From Warning to Info? More descriptive `help` (`long` too) would be good.